docs/swagger.yaml

```diff
@@ -2638,10 +2657,246 @@ paths:
+        '200':
+          $ref: '#/definitions/ChartInfoList'
+  /chartrepo/{repo}/charts/{name}:
+    get:
+      summary: Get all the versions of the specified chart
+      description: Get all the versions of the specified chart
+      tags:
+        - Products
+        - Chart Repository
+      parameters:
+        - name: repo
+          in: path
+          type: string
+          required: true
+          description: The project name
+        - name: name
+          in: path
+          type: string
+          required: true
+          description: The chart name
+      responses:
+        '401':
+          $ref: '#/definitions/UnauthorizedChartAPIError'
+        '403':
+          $ref: '#/definitions/ForbiddenChartAPIError'
+        '404':
+          $ref: '#/definitions/NotFoundChartAPIError'
+        '500':
+          $ref: '#/definitions/InternalChartAPIError'
+        '200':
+          $ref: '#/definitions/ChartVersions'
+  /chartrepo/{repo}/charts/{name}/{version}:
+    get:
+      summary: Get the specified chart version
+      description: Get the specified chart version
+      tags:
+        - Products
+        - Chart Repository
+      parameters:
+        - name: repo
+          in: path
+          type: string
+          required: true
+          description: The project name
+        - name: name
+          in: path
+          type: string
+          required: true
+          description: The chart name
+        - name: version
+          in: path
+          type: string
+          required: true
+          description: The chart version
+      responses:
+        '401':
+          $ref: '#/definitions/UnauthorizedChartAPIError'
+        '403':
+          $ref: '#/definitions/ForbiddenChartAPIError'
+        '404':
+          $ref: '#/definitions/NotFoundChartAPIError'
+        '500':
+          $ref: '#/definitions/InternalChartAPIError'
+        '200':
+          $ref: '#/definitions/ChartVersionDetails'
+    delete:
+      summary: Delete the specified chart version
+      description: Delete the specified chart version
+      tags:
+        - Products
+        - Chart Repository
+      parameters:
+        - name: repo
+          in: path
+          type: string
+          required: true
+          description: The project name
+        - name: name
+          in: path
+          type: string
+          required: true
+          description: The chart name
+        - name: version
+          in: path
+          type: string
+          required: true
+          description: The chart version
+      responses:
+        '401':
+          $ref: '#/definitions/UnauthorizedChartAPIError'
+        '403':
+          $ref: '#/definitions/ForbiddenChartAPIError'
+        '404':
+          $ref: '#/definitions/NotFoundChartAPIError'
+        '500':
+          $ref: '#/definitions/InternalChartAPIError'
+        '200':
+          description: The specified chart entry is successfully deleted.
+  /api/chartrepo/{repo}/charts:
```

Comment:
The `api` should be removed, same to `/api/chartrepo/{repo}/prov` and `/api/chartrepo/charts`.